canary/ClientApp/src/components/tests/FHIRIJEValidatorProducing.js

```diff
@@ -0,0 +1,216 @@
+              <Breadcrumb.Section as={Link} to="/">
+                Dashboard
+              </Breadcrumb.Section>
+              <Breadcrumb.Divider icon="right chevron" />
+              <Breadcrumb.Section>Validate FHIR Death Records (Producing)</Breadcrumb.Section>
+            </Breadcrumb>
+          </Grid.Row>
+          {!!this.state.test && this.state.test.completedBool && (
+            <Grid.Row className="loader-height">
+              <Container>
+                <div className="p-b-10" />
+                <Statistic.Group widths="three">
+                  <Statistic size="large">
+                    <Statistic.Value>{this.state.test.total}</Statistic.Value>
+                    <Statistic.Label>Properties Checked</Statistic.Label>
+                  </Statistic>
+                  <Statistic size="large" color="green">
+                    <Statistic.Value>
+                      <Icon name="check circle" />
+                      {this.state.test.correct}
+                    </Statistic.Value>
+                    <Statistic.Label>Correct</Statistic.Label>
+                  </Statistic>
+                  <Statistic size="large" color="red">
+                    <Statistic.Value>
+                      <Icon name="times circle" />
+                      {this.state.test.incorrect}
+                    </Statistic.Value>
+                    <Statistic.Label>Incorrect</Statistic.Label>
+                  </Statistic>
+                </Statistic.Group>
+                <Grid centered columns={1} className="p-t-30 p-b-15">
+                  <Button icon labelPosition='left' primary onClick={() => this.downloadAsFile(report(this.state.test, this.state.test.testId))}><Icon name='download' />Generate Downloadable Report</Button>
+                </Grid>
+                <div className="p-b-20" />
+                <Form size="large">
+                  <FHIRInfo fhirInfo={this.state.test.results} editable={false} testMode={true} updateFhirInfo={null} />
+                </Form>
+              </Container>
+            </Grid.Row>
+          )}
+          {!(!!this.state.test && this.state.test.completedBool) && (
+            <React.Fragment>
+              <Grid.Row>
+                <Container fluid>
+                  <Divider horizontal />
+                  <Header as="h2" dividing id="step-1">
+                    <Icon name="upload" />
+                    <Header.Content>
+                      Step 1: Upload IJE Record
+                      <Header.Subheader>
+                        Upload the IJE record you will use to validate the FHIR record. The below prompt allows you to paste the record, upload it as a file, or POST to Canary.
+                      </Header.Subheader>
+                    </Header.Content>
+                  </Header>
+                  <div className="p-b-15">
+                    <Getter name="upload-ije" id="upload1" updateRecord={this.updateRecord} ijeOnly noFormat />
```

Comment:
Normally when the user submits a file we show a success or fail message below so it is clear that they are able to continue. That may be useful on both the upload boxes.


![Screen Shot 2021-06-07 at 5 47 52 PM](https://user-images.githubusercontent.com/2308869/121091763-86897f80-c7b8-11eb-8dc7-88c46ff51d4b.png)